internal/contracts/contracts.go

```diff
@@ -274,4 +275,31 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.StateNonce,
 	}
 	return c, nil
+	
+}
+
+
+
+func GenerateRandomContract() Contract{
+	b := rand.Read(make([]byte, 32))
+	
+	genSenderPubKey := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	genrecipPubKeyHash := rand.Read(make([]byte, 32))
+	genVersion := binary.LittleEndian.Uint16(b[:])
+	genSigLen := binary.LittleEndian.Uint8(b[:]);
+	genSignature := rand.Read(make([]byte, genSigLen));
+	genValue := binary.LittleEndian.Uint64(b[:])
+	genStateNonce := binary.LittleEndian.Uint64(b[:])
+	
+	return Contract{
```

Comment:
Instead of defining the struct with `Contract { ... } `, leverage the `New` function instead that returns a contract, just feed it your `gen_` parameters.